modal/_utils/async_utils.py

```diff
@@ -31,6 +32,10 @@
 P = ParamSpec("P")
 V = TypeVar("V")
 
+if sys.platform == "win32":
+    # quick workaround for deadlocks on shutdown - need to investigate further
+    asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
```

Comment:
Changing the event loop policy feels like a major change.

Reply:
This inverts a synchronicity change in the other direction. Want to use the default one but it broke some test fixtures in modal client so reverted it "modal side" until we have time to look at that